.travis.yml

```diff
@@ -5,6 +5,7 @@ jdk:
 install:
  - mvn -version; docker version
 script:
+ - [[ $(git grep import | grep '\*' | grep -v '.mvn' | wc -l) == 0 ]]
```

Comment:
I know this is a WIP and so on but would this handle an echo?

`... == 0 || echo "Wildcard import found!"`

Reply:
IMO you can use this (without `[[ ]]`) when you test a variable. I need to execute bash command. However good point with echo a message. That is definitely needed!